apps/web/src/app/(networks)/(non-evm)/stellar/_common/lib/assets/tokens/mainnet/baseTokens.ts

```diff
@@ -29,36 +30,40 @@ export const baseTokens: Token[] = [
     name: 'SUSHI',
     org: 'sushi',
     domain: 'sushi.com',
+    icon: 'https://cdn.sushi.com/image/upload/f_auto,c_limit,w_48/d_unknown.png/tokens/1/0x6B3595068778DD592e39A122f4f5a5cF09C90fE2.jpg',
+    decimals: 7,
+  },
+  {
+    code: 'USDC',
+    issuer: 'GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN',
+    contract: 'CCW67TSZV3SSS2HXMBQ5JFGCKJNXKZM7UQUWUZPUTHXSTZLEO7SJMI75',
+    name: 'USDC',
+    org: 'centre',
+    domain: 'centre.io',
+    icon: 'https://stellar.expert/asset-img/GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN-USDC',
+    decimals: 7,
+    isStable: true,
+  },
+  {
+    code: 'EURC',
+    issuer: 'GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2',
+    contract: 'CDTKPWPLOURQA2SGTKTUQOWRCBZEORB4BWBOMJ3D3ZTQQSGE5F6JBQLV',
+    name: 'EURC',
+    org: 'centre',
+    domain: 'circle.com',
+    icon: 'https://stellar.expert/asset-img/GDHU6WRG4IEQXM5NZ4BMPKOXHW76MZM4Y2IEMFDVXBSDP6SJY4ITNPP2-EURC',
+    decimals: 7,
+  },
+  {
+    code: 'PYUSD',
+    issuer: 'GDQE7IXJ4HUHV6RQHIUPRJSEZE4DRS5WY577O2FY6YQ5LVWZ7JZTU2V5',
+    contract: 'CCCRWH6Q3FNP3I2I57BDLM5AFAT7O6OF6GKQOC6SSJNDAVRZ57SPHGU2',
+    name: 'PYUSD',
+    org: 'paxos',
+    domain: 'paxos.com',
+    icon: 'https://stellar.expert/asset-img/GDQE7IXJ4HUHV6RQHIUPRJSEZE4DRS5WY577O2FY6YQ5LVWZ7JZTU2V5-PYUSD',
```

Comment:
I'm hesitant to add more tokens until we remove the fallback to query pools by brute-force.

Particularly worried about RPC usage here using up the rate-limit for other more critical calls.

Reply:
I don't think this is something we can mitigate until there's a better option to fetch these; if the sushi team wants it in that's something we'll make clear to them.